src/TextCounter.cpp

```diff
@@ -1,9 +1,10 @@
 #include "TextCounter.h"
+#include "ResourceManager.h"
 
 TextCounter::TextCounter(float x, float y, const char* text, sf::Color color):
     counterName{text}
 {
-    font.loadFromFile("Res/consola.ttf");
+    font = ResourceManager::Get().GetFont("Text");
```

Comment:
just like with the texture in the SpriteComponent, we don't want to have the font here.
`counterText.setFont(ResourceManager::Get().GetFont("Text"));`